### DISCUSSION TITLE
Inicio - criando pasta

### DISCUSSION THREAD

**Discussion by Alexmau10** [balta-io/2801#43] [Perguntas e Respostas] [answered]
Olá @andrebaltieri, Muito prazer e parabéns pela ótima didática!
Estou com uma dificuldade logo no início do 'hands on' ... por que não consigo entrar na pasta dev que criei no c: para logo em seguida fazer o mkdir Teste lá dentro ?

![image](https://user-images.githubusercontent.com/89946503/153417160-9cbba247-bfec-4f60-9aec-7d4561b0f6eb.png)

![image](https://user-images.githubusercontent.com/89946503/153417264-652e3457-669d-4add-8a7f-59182bef04a9.png)

**Comment by niziul** [comment]
@Alexmau10 

O comando que cê tá tentando usar não funciona.

Para conseguir navegar pelas pastas no powerShell é usado os seguintes comandos: `cd .\nomeDaPasta\` ou ao digitar `cd .\` use as seguintes teclas: "Crtl" + "Espaco" que será exibido os possíveis nomes presentes na pasta que podem ser acessada, ou `cd ..` para retroceder uma pasta. 

+ Exemplo visual:
![](https://i.ibb.co/0q2YGSQ/imagem-2022-02-10-164341.png)

**Reply by Dallonso** [reply]
Boa noite @Alexmau10 , tudo certo?

Pelo seu print, você esta na pasta "Alexandre" (C:\Users\Alexandre)

O correto seria você voltar duas pastas até a "C:" usando o comando cd.. e depois navegar para dentro da pasta dev:

> C:\Users\Alexandre>**_cd .._**
> C:\Users>**_cd .._**
> C:\>
> C:\>**_cd dev_**
> C:\dev>

Não sei se é possível voltar da pasta Alexandre para a C: de algum modo direto, acredito que não.

**Reply by Alexmau10** [reply]
Perfeito agora! Grato Dallonso!

**Comment by marcelobarbieri** [comment, marked as answer]
Boa noite a todos.

Olá Diego,

É possível voltar diretamente para o C:\
Veja a imagem em anexo.

Abraços

[image: image.png]

Em qui., 10 de fev. de 2022 às 21:06, Diego Alonso ***@***.***>
escreveu:

> Boa noite @Alexmau10 <https://github.com/Alexmau10> , tudo certo?
>
> Pelo seu print, você esta na pasta "Alexandre" (C:\Users\Alexandre)
>
> O correto seria você voltar duas pastas até a "C:" usando o comando cd.. e
> depois navegar para dentro da pasta dev:
>
> C:\Users\Alexandre>*cd ..*
> C:\Users>*cd ..*
> C:>
> C:>*cd dev*
> C:\dev>
>
> Não sei se é possível voltar da pasta Alexandre para a C: de algum modo
> direto, acredito que não.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/balta-io/2801/discussions/43#discussioncomment-2154098>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUTJXVUJ7QQPMRQPPQV6R73U2RHKDANCNFSM5OAXLEYA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Marcelo de Paiva Barbieri

**Comment by andrebaltieri** [comment]
@Alexmau10 faltou o `cd`...
O comando correto é `cd C:\dev`

**Reply by Alexmau10** [reply]
Perfeito agora! Grato Balta!

**Comment by Alexmau10** [comment]
@NiziulLuizin @andrebaltieri @Dallonso 
Obrigado Pessoal!

**Comment by Alexmau10** [comment]
![image](https://user-images.githubusercontent.com/89946503/153620746-0cfd07e5-6b00-4505-81b0-5bf4b23248be.png)

**Reply by andrebaltieri** [reply]
Parabéns 💜
